css/main.css

```diff
@@ -621,17 +621,17 @@ main.content {
 
 .book-list-column {
   float: left;
-  width: 10.25em;
+  width: 8.4em;
```

Comment:
This can be tricky with different languages, i.g Russian
![Знімок екрану з 2021-09-16 07-31-38](https://user-images.githubusercontent.com/3584119/133612685-0c9e464e-16df-4ed9-acee-55c6f6e24031.png)